src/Microsoft.Security.Utilities.Core/CommonAnnotatedKey.cs

```diff
@@ -59,8 +59,10 @@ public static bool TryCreate(string key, out CommonAnnotatedKey secret)
             return true;
         }
 
-        private byte[] bytes;
-        private string base64Key;
+        protected byte[] bytes;
```

Comment:
>protected [](http://example.com/codeflow?start=8&length=9)

protected fields actually aren't legal according to the .NET guidelines!

Instead, create properties, can we just make the setter protected?

public string EncodedKey { get; protected set; }
public byte[] KeyBytes { get; protected set; }

Reply:
got it

Reply:
updated to properties with protected set